backend/src/Core/FurnitureShop.Core.Domain/Category.cs

```diff
@@ -1,15 +1,26 @@
 using System;
+using System.Collections.Generic;
+using LeanCode.DomainModels.Model;
+using LeanCode.Time;
 
 namespace FurnitureShop.Core.Domain
 {
-    public class Category
+    public class Category : IAggregateRoot<Id<Category>>
     {
+        public Id<Category> Id { get; set; }
+        public string Name { get; set; }
+        byte[] IOptimisticConcurrency.RowVersion { get; set; } = Array.Empty<byte>();
+        DateTime IOptimisticConcurrency.DateModified { get; set; }
         public Category(string name)
         {
-            this.Name = name;
+            Name = name;
+            Id = Id<Category>.New();
         }
 
-        public Guid Id { get; set; }
-        public string Name { get; set; }
+        public Category()
```

Comment:
Just remove this time of constructors with incomplete info, for not create empty strings, it will solve a lot of problems with database